demisto_sdk/commands/common/hook_validations/release_notes.py

```diff
@@ -120,6 +120,25 @@ def is_docker_image_same_as_yml(self) -> bool:
 
         return True
 
+    def is_breaking_change(self) -> bool:
+        """
+        In case of a breaking change in the release note, ensure the existence of a proper json file.
+        """
+        is_valid = True
+        if 'breaking change' in self.latest_release_notes.lower():
```

Comment:
what will happen in this line if there is no release notes file?

Reply:
What do you mean by *there is no release notes file*?
This section will run only if there are release note else it will fail by the main class of the SDK.

Reply:
I mean, what happens if `self.latest_release_notes` is `None` - lower() fill fail.
If I understand correctly, `is_breaking_change()` runs before this check (`has_release_notes_been_filled_out()`).